remote/.yarnrc

```diff
@@ -1,3 +1,3 @@
 disturl "http://nodejs.org/dist"
-target "12.4.0"
+target "12.14.1"
```

Comment:
@alexdima fyi not sure why `remote` locked the dependency to `12.4.0` but it would now advance to the same as Electron 9

Reply:
:+1: Maybe there was no node release with the same version number as the one used in Electron?